src/app/devices/alarms/alarms.component.html

```diff
@@ -14,11 +14,16 @@
             <div class="flex-75">
               <span matListItemTitle data-cy="alarmElementName">{{entry.ElementName}} (InstanceID:
               {{entry.InstanceID}})</span>
-              <span matListItemLine data-cy="alarmStartTime">Start Time: {{entry.StartTime.Datetime | date:
-              'short'}}</span>
+              @if(cloudMode) {
```

Comment:
We shouldnt need the condition if MPS and Console are API compatible right?

Reply:
Both MPS and Console, give almost same output except in console we have additional field "IntervalInMinutes". Also, how we parse the data changes little bit.